contracts/Makefile

```diff
@@ -0,0 +1,23 @@
+.PHONY: all deps compile clean generate-abi
+
+all: deps compile lint generate-abi
+
+#=============================================================================#
+#                            Smart Contracts                                  #
+#=============================================================================#
+
+compile:
+	@forge compile
+
+clean:
+	@forge clean && \
+	rm -rf node_modules/
+
+lint:
+	@forge fmt
+
+deps:
+	@bun install
+
+generate-abi:
+	@jq '.abi' ./out/OrbiterGatewayCCTP.sol/OrbiterGatewayCCTP.json > ./abi/OrbiterGatewayCCTP.abi
```

Comment:
nit: `generate-abi` is misleading here because the compilation already needs to have happened at this point -- maybe rather call it `extract-abi`?